scripts/barclamp_mgmt_lib.rb

```diff
@@ -488,11 +455,6 @@ def bc_install_layout_1_app(from_rpm, bc, bc_path, yaml)
     FileUtils.rm manifest_created rescue nil
   end
 
-  filelist = File.join BARCLAMP_PATH, "#{bc}-filelist.txt"
-  File.open( filelist, 'w' ) do |out|
-    files.each { |line| out.puts line }
-  end
```

Comment:
Note that the uninstall command requires this file to exist (so it can remove files that were copied by the install script).